src/main/resources/templates/openingShow.html

```diff
@@ -97,7 +97,7 @@ <h3 th:text="#{lang.export.title}">Export</h3>
 			<form th:action="@{/configure/export}" method="get">
 				<span class="note" style="margin-bottom: 0" th:text="#{lang.export.scheme}">Select scheme:</span>
 				<select name="type">
-					<option th:each="et : ${exportTypes}" th:object="${et}" th:value="*{name()}" th:text="*{displayName}">Dzsájrosz</option>
+					<option th:each="et : ${exportTypes}" th:object="${et}" th:value="*{name}" th:text="*{displayName}">Dzsájrosz</option>
```

Comment:
Thymeleaf is still a Java based engine, so an enum's name should be reached with name(), right? (Because it didn't work when I tried)